packages/shared/src/components/feeds/FeedNav.tsx

```diff
@@ -82,6 +100,17 @@ function FeedNav(): ReactElement {
             className: { indicator: '!w-6', item: 'px-1' },
             autoScrollActive: true,
           }}
+          renderTab={({ label }) => {
+            if (label === FeedNavTab.NewFeed) {
+              return (
+                <div className="flex size-6 items-center justify-center rounded-6 bg-background-subtle">
+                  <PlusIcon />
+                </div>
+              );
+            }
+
+            return null;
+          }}
```

Comment:
Had to add support to render custom tab due to "+" icon

<img width="451" alt="image" src="https://github.com/dailydotdev/apps/assets/9803078/3c314ea1-7fec-4755-a7ae-c4fd8fead5b9">